### PR TITLE
Small improvements to code

### PR DESCRIPTION
**Pull request by imuchnik**
Short description explaining the high-level reason for the pull request

## Additions
-

## Removals
- Removal of getters

## Changes
- Formatting
- Fix the json annotation to no ever marshal out the password

## Review

- @phungocnguyen 

[Preview this PR without the whitespace changes](?w=0)

### CODE REVIEW

File: parse_Excel_loadTo_Sql.go [hunk 1]
```diff
@@ -119,15 +95,15 @@ func getConfig(configFile string) Config {
 func getConnStr(config Config) string {
```
Comment (marcesher) [inline parse_Excel_loadTo_Sql.go:96]: I'd go even further here. Get rid of the buffer.WriteString stuff and just do it with a single `fmt.Sprintf()`. See https://golang.org/pkg/fmt/

File: parse_Excel_loadTo_Sql.go [hunk 2]
```diff
@@ -80,8 +56,8 @@ func main() {
 		log.Fatal(err)
 	}
 
-	fmt.Printf("Set to schema [%v]\n", config.GetSchema())
-	setSearchPath(db, config.GetSchema())
+	fmt.Printf("Set to schema [%v]\n", config.Schemago)
```
Comment (marcesher) [inline parse_Excel_loadTo_Sql.go]: `Schemago` looks wrong here. Surprised it even compiled for you

Comment (imuchnik) [reply parse_Excel_loadTo_Sql.go]: Yep. Good catch @marcesher.

File: parse_Excel_loadTo_Sql.go (hunk 1 above)
Comment (imuchnik) [reply parse_Excel_loadTo_Sql.go:96]: Something like this:
```
fmt.Sprintf("postgres://%s:%s@%s/%s?sslmode=",
		config.Username,config.Password,config.Location, config.Dbname,config.Sslmode)
```

?

**Pull request merged**